dict/prh-idiomatic-usage.yml

```
@@ -908,6 +908,15 @@ rules:
         to: 再読み込み
       - from: 再読み込み
         to: 再読み込み
+  - expected: ウィンドウ
+    pattern:
+      - ウインドウ
+    prh: >-
+      「ウィ/ウェ/ウォ」を使用する
+      https://smarthr.design/products/contents/idiomatic-usage/usage/#recvhB6VJstpBQKeV-0
```

@kuvanov-2 
[こちら](https://kufuinc.slack.com/archives/C01L7GK9APK/p1673412404946069?thread_ts=1673314377.407019&cid=C01L7GK9APK)を踏まえてsuggestしました！
```suggestion
      https://smarthr.design/products/contents/idiomatic-usage/usage/
```